src/middleware/err_handlers.rs

```diff
@@ -37,27 +37,21 @@ type ErrorHandler<B> = dyn Fn(ServiceResponse<B>) -> Result<ErrorHandlerResponse
 ///
 /// # Examples
 /// ```
-/// use actix_web::middleware::{ErrorHandlers, ErrorHandlerResponse};
-/// use actix_web::{web, dev, App, HttpRequest, HttpResponse, Result};
-/// use actix_web::http::{StatusCode, header};
-///
-/// fn render_500<B>(mut res: dev::ServiceResponse<B>) -> Result<ErrorHandlerResponse<B>> {
-///     res.response_mut()
-///        .headers_mut()
-///        .insert(header::CONTENT_TYPE, header::HeaderValue::from_static("Error"));
```

Comment:
I copied this into a non-rustdoc setting and let `rustfmt` reformat it to what you see below.